crates/bitcoin-da/src/helpers/builders.rs

```diff
@@ -184,9 +187,16 @@ fn build_commit_transaction(
     } else {
         (None, utxos)
     };
+    let mut iteration = 0;
     let mut last_size = size;
 
     let tx = loop {
+        if iteration % 100 == 0 {
+            trace!(iteration, "Trying to find commitment size");
+            if iteration > 1000 {
```

Comment:
1000 is too low for this imo

go for 5k maybe? 10k?

Reply:
Sure